run_tests.sh

```diff
@@ -2,9 +2,20 @@
 
 set -ex
 
-# run tests
-env GORACE="halt_on_error=1" go test -race -short ./...
+dir=$(pwd)
+# list of all modules to test
+modules=". client/cmd/dexcctl"
 
+# Test each module separately.
+for m in $modules
+do
+	cd $dir/$m
+	# run tests
+	env GORACE="halt_on_error=1" go test -race -short ./...
+done
+
+# Return to initial directory.
+cd $dir
 # golangci-lint (github.com/golangci/golangci-lint) is used to run each
 # static checker.
 
```

Comment:
It didn't look like the unit tests for dexcctl were being run. This works but not sure if it is the right way to do this.
@dajohi Is this necessary?

Reply:
I don't know if this is the "right" way, but it looks fine to me for checking explicitly listed modules.